### DISCUSSION TITLE
redesign do editor-emendas?

### DISCUSSION THREAD

**Discussion by ghost** [lexml/editor-emendas#130] [Q&A] [answered]
Oi pessoal.

*ideia*
Gostaria de saber o que pensam desse redesign do editor-emendas.

*img-antes*
![image](https://user-images.githubusercontent.com/111701513/212804774-42b6da7e-dc14-4522-a2c5-56073a8b7fb2.png)

*img-depois*
![image](https://user-images.githubusercontent.com/111701513/212804610-ece0ca91-ea15-4f86-b073-0783d8f61524.png)

*conceito*
- Minha ideia seria que o editor-emendas fosse próximo ao google docs para facilitar a experiência do usuário.
- Comandos, atalhos e dicas podem ser feitos com um tooltip. Dessa forma o usuário vai conhecer a ferramenta passo a passo, de forma guiada pela própria ferramenta de como utilizar a ferramenta.
- Esse redesign foi feito baseado na ferramenta [editor-js](https://editorjs.io/), uma alternativa ao editor-js é o [lexical-js](https://playground.lexical.dev/). Ambos são editores de texto em bloco, algo semelhante ao Notion e ferramentas como Google Docs. Ambos projetos de código fonte aberto e poderiam ser utilizados para melhorar ainda mais a experiência de uso para qualquer usuário ou pessoa no editor-emendas.

*feedback*
Gostaria de saber o que pensam dessa ideia - essa ideia é boa ou é ruim?

**Comment by arthurcordeiro** [comment, marked as answer]
Olá @codehangen,

A ideia de aproximar a experiência do usuário ao google docs é um objetivo nosso também. Agradecemos sua sugestão.

Se tiver disponibilidade em colaborar conosco gostaria de conversar contigo.  Tendo interesse envie a melhor data/hora para o e-mail arthur.cordeiro@gmail.com.

Obrigado.

**Reply by ghost** [reply]
Oi arthurcordeiro.

> Olá @codehangen,
> A ideia de aproximar a experiência do usuário ao google docs é um objetivo nosso também. Agradecemos sua sugestão.
> Se tiver disponibilidade em colaborar conosco gostaria de conversar contigo. Tendo interesse envie a melhor data/hora para o e-mail [arthur.cordeiro@gmail.com](mailto:arthur.cordeiro@gmail.com).
> Obrigado.

fico feliz com teu feedback.
